pkg/integration/components/view_driver.go

```diff
@@ -403,6 +403,14 @@ func (self *ViewDriver) PressFast(keyStr string) *ViewDriver {
 	return self
 }
 
+func (self *ViewDriver) Click(x, y int) *ViewDriver {
+	offsetX, offsetY, _, _ := self.getView().Dimensions()
+
+	self.t.click(offsetX+1+x, offsetY+1+y)
```

Comment:
@jesseduffield I was unsure why the additional `+1`s are needed here, but they are. I expected `offsetX+x, offsetY+y` to work here. Just curious what the intuition is that I'm missing here.

Reply:
I assume that's due to the frame of the view. Although if it also applies for views without frames (e.g. the information view) then I too am confused.